Makefile.autosetup

```diff
@@ -424,6 +424,17 @@ distclean: clean
 	$(RM) $(DEPFILES) .clang_complete autosetup/jimsh0 config.h config.log \
 		conststrings.c contrib/Makefile doc/Makefile doc/neomutt.1 \
 		html Makefile po/Makefile test/Makefile
+	$(RM) *.gc?? */*.gc?? test/*/*.gc??
+	$(RM) lcov.info lcov
+
+# coverage testing
+force:
```

Comment:
?

Reply:
The `lcov` target creates an `lcov` directory.
The `force` target means the `lcov` target will always be rebuilt when asked for.

I suppose I could change the target to `coverage` and drop `force:`

Reply:
Oh, I think .PHONY would be clearer in this case, but yeah, let's call that `coverage`